dev-start.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
```

Comment:
Can this script check that docker is installed on the dev's machine and advise how to install it if it is not?

E.g

```
type docker >& /dev/null || ( echo 'docker required' && exit 1 )
```

It could even go further and if it does not find it install docker itself.

Reply:
👍 Updated now. We're now checking the requirements listed [here](https://github.com/guardian/grid/blob/master/docs/02.01-dev-setup.md#requirements)